lib/dust.js

```diff
@@ -897,14 +856,6 @@
     }
   };
 
-  if (typeof define === "function" && define.amd && define.amd.dust === true) {
-    define("dust.core", function() {
-      return dust;
-    });
-  } else if (typeof exports === 'object') {
-    module.exports = dust;
-  } else {
-    root.dust = dust;
-  }
+  return dust;
 
-})((function(){return this;})());
```

Comment:
I've tried to fix this so many times...This was for `console.log` according to blame (99b5cede91bbce196046910ce701a6eedd290f56).